projects/cashmere/src/lib/form-field/hc-form-field.component.html

```diff
@@ -1,4 +1,9 @@
-<div [class.hc-form-field-wrapper-inline]="inline" [class.hc-form-field-wrapper]="!inline" [class.hc-form-field-no-label]="!hasLabel">
+<div
+    [class.hc-form-field-wrapper-inline]="inline"
```

Comment:
I'm actually not sure what this binding does. Mind explaining that to me?

Reply:
I think it connects to that `@HostBinding` that sets a field based on whether the class is present and this is a way to then communicate that or something?